pkg/sql/compile/sql_executor_context.go

```diff
@@ -85,7 +85,11 @@ func (c *compilerContext) Stats(obj *plan.ObjectRef) bool {
 	if err != nil {
 		return false
 	}
-	return t.Stats(c.ctx, nil, c.GetStatsCache().GetStatsInfoMap(t.GetTableID(c.ctx)))
+	s := c.GetStatsCache().GetStatsInfoMap(t.GetTableID(c.ctx), true)
```

Comment:
Under what circumstances would this s be nil?

Reply:
before refactor, this api never return nil. after refactor, it could return nil.